robottelo/manifests.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, content=None, filename=None):
         if self._content is None:
             self._content = _manifest_cloner.clone()
         if self.filename is None:
-            self.filename = u'/tmp/manifest-{0}.zip'.format(int(time.time()))
+            self.filename = u'/var/tmp/manifest-{0}.zip'.format(int(time.time()))
```

Comment:
The changes are good but this is making flake8 not happy. Will merge the PR when flake8 is happy again.